### PR TITLE
[TI-2447] Golang version upgrade in dockerfiles

### PR DESCRIPTION
**Pull request by Rajesh-enlume**

### CODE REVIEW

File: 1.17-alpine-fuzz/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM golang:1.17-alpine
+
+RUN apk add --no-cache gcc musl-dev git bash
+RUN wget -O- -nv https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.38.0
+RUN mv /go/bin/golangci-lint /usr/local/bin/
+RUN wget -O /usr/local/bin/swagger "https://github.com/go-swagger/go-swagger/releases/download/v0.26.1/swagger_linux_amd64"
```
Comment (JSchwerberg) [inline 1.17-alpine-fuzz/Dockerfile]: Let's update to https://github.com/go-swagger/go-swagger/releases/tag/v0.29.0

File: 1.17-alpine-fuzz/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+FROM golang:1.17-alpine
+
+RUN apk add --no-cache gcc musl-dev git bash
+RUN wget -O- -nv https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.38.0
```
Comment (JSchwerberg) [inline 1.17-alpine-fuzz/Dockerfile]: https://github.com/golangci/golangci-lint/releases/tag/v1.44.0

File: 1.17-alpine-fuzz/Dockerfile (hunk 1 above)
Comment (Rajesh-enlume) [reply 1.17-alpine-fuzz/Dockerfile]: Updated the url

File: 1.17-alpine-fuzz/Dockerfile (hunk 2 above)
Comment (Rajesh-enlume) [reply 1.17-alpine-fuzz/Dockerfile]: updated.

File: 1.17-alpine/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+RUN wget https://github.com/golangci/golangci-lint/releases/download/v1.44.0/golangci-lint-1.44.0-linux-amd64.tar.gz
+RUN tar -xzf golangci-lint-1.44.0-linux-amd64.tar.gz
+RUN mv golangci-lint-1.44.0-linux-amd64/golangci-lint /usr/local/bin
+RUN chmod +x /usr/local/bin/golangci-lint
+RUN wget -O /usr/local/bin/swagger "https://github.com/go-swagger/go-swagger/releases/download/v0.29.0/swagger_linux_amd64"
+RUN chmod +x /usr/local/bin/swagger
+RUN wget "https://github.com/ory/go-acc/releases/download/v0.2.7/go-acc_0.2.7_linux_amd64.tar.gz"
+RUN tar -xzf go-acc_0.2.7_linux_amd64.tar.gz
+RUN mv go-acc /usr/local/bin
+RUN chmod +x /usr/local/bin/go-acc
```
Comment (JSchwerberg) [inline 1.17-alpine/Dockerfile]: Nit: Add newline

**Review by JSchwerberg — APPROVED** [review]

File: 1.17-alpine/Dockerfile (hunk 3 above)
Comment (Rajesh-enlume) [reply 1.17-alpine/Dockerfile]: Added newline.

**Pull request merged**